lib/hashie/extensions/persistable.rb

```diff
@@ -0,0 +1,38 @@
+  module Hashie
+    module Extensions
+      module Persistable
+        CANNOT_INCLUDE = 'Peristable can only be used on classes with Hashie::Extensions::HashInitializer'
```

Comment:
If we want text in exceptions, I think I want something evolved like [this](http://code.dblock.org/2013/05/15/being-helpful-with-ruby-exceptions-and-error-messages.html).